React/Views/RCTMap.h

```diff
@@ -26,10 +26,12 @@ extern const CGFloat RCTMapZoomBoundBuffer;
 @property (nonatomic, assign) UIEdgeInsets legalLabelInsets;
 @property (nonatomic, strong) NSTimer *regionChangeObserveTimer;
 @property (nonatomic, copy) NSArray<NSString *> *annotationIds;
+@property (nonatomic, strong) NSMutableArray *polylineIds;
```

Comment:
This property should be publicly immutable, like annotationIds. It's generally poor practice to expose mutable collections unless you absolutely have to.